Calibration/Hotline/python/hotlineSkims_Output_cff.py

```diff
@@ -0,0 +1,33 @@
+import FWCore.ParameterSet.Config as cms
+import copy
+
+from Configuration.EventContent.EventContent_cff import FEVTEventContent
+
+OutALCARECOHotline = cms.PSet(
```

Comment:
this looks like github web artifact: #9802 code appears as a part of this PR for some reason (probably an unresolved direction of a merge commit).
Can this be cleaned up?